system/systemd.py

```diff
@@ -225,13 +232,16 @@ def main():
                 enabled = dict(type='bool'),
                 masked = dict(type='bool'),
                 daemon_reload= dict(type='bool', default=False, aliases=['daemon-reload']),
+                user= dict(type='bool', default=False),
             ),
             supports_check_mode=True,
             required_one_of=[['state', 'enabled', 'masked', 'daemon_reload']],
         )
 
     # initialize
     systemctl = module.get_bin_path('systemctl')
+    if module.params['user']:
+        systemctl = systemctl + " --user"
```

Comment:
@bcoca just by googling around I found https://www.archlinux.org/news/d-bus-now-launches-user-buses/ and  http://forum.armbian.com/index.php/topic/470-systemctl-user-failed-to-get-d-bus-connection-connection-refused/ and seems X isn't a requirement AFAICT, maybe systemd version?